eth_account/account.py

```diff
@@ -56,23 +56,23 @@
 
 
 class Account(object):
-    '''
-    This is the primary entry point for working with Ethereum private keys.
+    """
+    Account is the primary entry point for working with Ethereum private keys.
```

Comment:
Pretty nitpicky but...maybe just remove "Account is" from the beginning and capitalize the "t" in "the"?  Or, use the double-backtick monospace formatting for signify that "Account" refers to a class and isn't just another word that begins the sentence.  But I don't like that because then the name of the class could potentially get out of sync with the documentation.